src/codesyncd/websocket/socket_events.ts

```diff
@@ -97,6 +111,23 @@ export class SocketEvents {
             }
             this.connection.send(JSON.stringify({"diffs": validDiffs}));
         }
+
+        // Send tabs
+        let validTabs: ITabYML[] = [];
+        errorCount = 0;
+        if (!this.repoTabs) return;
+        for (const repoTab of this.repoTabs) {
+            const tabsHandler = new TabsHandler(repoTab, this.accessToken);
+            const tabs = await tabsHandler.run();
+            if (!tabs) return;
+            validTabs = validTabs.concat(tabs);
+        }
+
+        if (validTabs.length) {
+            CodeSyncLogger.debug(`Sending ${validTabs.length} tabs`);
+            this.connection.send(JSON.stringify({"tabs": validTabs}));
```

Comment:
We should call `sendTabsToServer()` here, right? Where are we using `sendTabsToServer()`?

Reply:
Good catch, fixing